data/tokens.json

```diff
@@ -3407,6 +3351,63 @@
         "textCase": "{gtmds_text-case-none}"
       },
       "type": "typography"
+    },
+    "gtmds_popover_margin": {
+      "value": "{gtmds_space-xs}",
+      "type": "spacing"
+    },
+    "gtmds_popover-header_margin-horizontal": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_popover-header_margin-vertical": {
+      "value": "{gtmds_space-xxs}/4",
+      "type": "spacing"
+    },
+    "gtmds_popover-footer_margin-horizontal": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_popover-body_margin-vertical": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_popover-body_color": {
+      "value": "{Text.gtmds_color_text-color}",
+      "type": "color"
+    },
+    "gtmds_popover-header_color": {
+      "value": "{Text.gtmds_color_text-color}",
+      "type": "color"
+    },
+    "gtmds_popover-header-subtitle_color": {
+      "value": "{Text.gtmds_color_text-color-subdued}",
+      "type": "color"
+    },
+    "gtmds_popover-body": {
+      "value": {
+        "fontFamily": "{gtmds_font-family-p}",
+        "fontWeight": "{gtmds_font-weight-400}",
+        "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
+        "lineHeight": "{gtmds_line-height-lg}",
+        "fontSize": "{Paragraph Size.gtmds_font-size-md}"
+      },
+      "type": "typography"
+    },
+    "gtmds_popover_background-color": {
+      "value": "{gtmds_color_gray-1}",
+      "type": "color"
+    },
+    "gtmds_popover-body-title": {
+      "value": {
+        "fontFamily": "{gtmds_font-family-p}",
+        "fontSize": "{Paragraph Size.gtmds_font-size-sm}"
+      },
+      "type": "typography"
+    },
```

Comment:
I don't see you using this one anywhere in Figma, it looks no to be applied to anything in the popover body, if that is the case we can remove the token. If it is used somewhere but I didn't notice I guess it would need to be subtitle instead of title to match the colors' naming.

Reply:
Removed it

Reply:
this was meant to be used for the body title on assignments.

Reply:
we decided to do 1 and use default icon button styles for 2 so I am resolving this. On my end all is good but feel free to add more typographies if needed.